web_client/templates/panels/annotationSelector.pug

```diff
@@ -67,14 +67,16 @@ block content
                   download=name + '.json')
                 span.icon-download.h-download-annotation(
                     title='Download annotation')
-
+  
   .checkbox.h-annotation-toggle
     label(title='Show annotation labels when hovering with the mouse.')
       input#h-toggle-labels(type='checkbox', checked=showLabels)
       | Labels
     label(title='Highlight annotations when hovering with the mouse.')
       input#h-toggle-interactive(type='checkbox', checked=interactiveMode)
       | Interactive
-    button.btn.btn-sm.btn-primary.h-create-annotation
-      | #[span.icon-plus-squared] New
+    if user && user.get 
+      if accessLevel > 0
```

Comment:
I'd rather not have hard-coded constants, since we define this.  It would be better to have this read `if accessLevel >= AccessType.WRITE`, which means that you'll have to pass `AccessType` when you invoke the template.

Reply:
Got it.